app/styles/_tables.less

```diff
@@ -151,8 +151,8 @@
   th#severity {width:10px;padding:0;}
   th#reason {width:150px;}
   th#message   {width:100%;} // ensures it gets remaining width
-  th#kind-name,th#message  {
-    .word-break;
```

Comment:
[merge]

On Thu, Aug 11, 2016 at 10:31 AM, Sam Padgett <notifications@github.com>
wrote:

> In app/styles/_tables.less
> <https://github.com/openshift/origin-web-console/pull/348#discussion_r74433017>
> :
>
> > @@ -151,8 +151,8 @@
> >    th#severity {width:10px;padding:0;}
> >    th#reason {width:150px;}
> >    th#message   {width:100%;} // ensures it gets remaining width
> > -  th#kind-name,th#message  {
> > -    .word-break;
>
> See openshift/origin#7883 <https://github.com/openshift/origin/pull/7883>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/348/files/689f69b43b384262afba5ef9ac401b330a264b1c#r74433017>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABZk7URZPIrabU7MO_OxzGfQ2WoYLtbLks5qezJVgaJpZM4JiKYf>
> .
>